testbed/core/models/auth.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+class TesterUser(AbstractUser):
```

Comment:
This seems unecessary - there's no part of TesterUser that's not already handled by the django User class.

* there's already email and email_verified
* we know if the "is_staff" field is not set then it's a tester
*

I worry this sets us up for a lot more work making things work in the future whereas django User object just works out of the box

Reply:
Yes, I wanted to talk about it.

I chose to implement a custom `TesterUser` model because I wanted to use email-based authentication and force email as the primary identifier. This also removed username requirement for cleaner signup flow. And lastly, the is_tester makes the user’s role explicit in our codebase.

But I completely understand the unnecesarry customization. We can work with Django’s built-in User model if that’s seems better. I know that would reduce maintenance overhead and keep the authentication standard.

And yes, we can avoid potential compatibility issues later on by using it.